packages/data-context/test/unit/sources/migration/autoRename.spec.ts

```diff
@@ -393,6 +393,60 @@ describe('applyMigrationTransform', () => {
       expect(result.before).to.eql(expected.before)
       expect(result.after).to.eql(expected.after)
     })
+
```

Comment:
Nice. Sorry to slow you down but can we add an additional integration test in `packages/launchpad/cypress/e2e/migration.cy.ts` to verify this works? Ideally we'd add a new project to `system-tests`, maybe called `migration-already-migrated` that showcases this workflow.

What do you think? Too heavy handed? Main concern is around quality here, I think this unit test covers the use case, but I want to be sure (eg, it doesn't verify what happens when you hit rename, or skip, etc, for an already migrated project).

Reply:
great idea !

Reply:
done

Reply:
Can you update the system test to actually click through the entire migration flow? Also, don't forget they can try to skip renaming, etc. Currently it is basically just a UI test.

Reply:
I updated the rest of the test, though its value seems to be lower.